libraries/core-react/src/components/Table/Table.test.tsx

```diff
@@ -41,3 +55,72 @@ describe('Caption', () => {
     )
   })
 })
+
+describe('Table', () => {
+  it('Can render a cell as a header cell', () => {
+    const text = 'Name'
+    const { getByText, container } = render(
+      <Table>
+        <Head>
+          <Row>
+            <Cell as="th">{text}</Cell>
+          </Row>
+        </Head>
+      </Table>,
+    )
+
+    const headerCell = getByText(text)
+    const th = container.querySelector('th')
+    expect(headerCell).toEqual(th)
+  })
+
+  it('Renders the header row visually different than the other rows by using a background colour', () => {
+    const header = 'Header'
+    const body = 'Body content'
+    const { getByText } = render(
+      <Table>
+        <Head>
+          <Row>
+            <Cell as="th">{header}</Cell>
+          </Row>
+        </Head>
+        <Body>
+          <Row>
+            <Cell>{body}</Cell>
+          </Row>
+        </Body>
+      </Table>,
+    )
+    const headElement = getByText(header).closest('thead')
+    const regularContent = getByText(body)
+    expect(headElement).not.toHaveStyleRule('background', 'rgba(255,255,255,1)')
+    expect(regularContent).toHaveStyleRule('background', 'rgba(255,255,255,1)')
```

Comment:
If I am not mistaken the Table head also has a thicker bottom border that can be asserted on.

Reply:
I added that as well. Using the tokens as assertion was just too complicated so I just added the border as a string. But things are slightly better by the usage of tokens for the white colour at least